pyrefly/lib/commands/check.rs

```diff
@@ -487,6 +487,63 @@ impl Args {
         )
     }
 
+    pub fn run_once_with_snippet(
+        self,
+        code: String,
+        _files_to_check: FilteredGlobs,
+        config_finder: ConfigFinder,
+        allow_forget: bool,
+    ) -> anyhow::Result<(CommandExitStatus, usize)> {
+        use std::path::PathBuf;
+        use std::sync::Arc;
+
+        use pyrefly_python::module_name::ModuleName;
+
+        use crate::module::module_path::ModulePath;
+
+        let mut timings = Timings::new();
```

Comment:
I don't think we need to time the setup here; we do so in `run_once` because listing files can take a while, but here there's nothing time-consuming happening.